pkg/services/tree/sync.go

```diff
@@ -0,0 +1,98 @@
+package tree
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/pilorama"
+	"github.com/nspcc-dev/neofs-node/pkg/network"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	"google.golang.org/grpc"
+)
+
+// Synchronize tries to synchronize log starting from the last stored height.
+func (s *Service) Synchronize(ctx context.Context, cid *cid.ID, treeID string) error {
+	nodes, err := s.getContainerNodes(cid)
+	if err != nil {
+		return err
```

Comment:
wrap with some additional info to help to understand that exactly went wrong?

Reply:
Fixed.